Makefile

```diff
@@ -1,41 +1,48 @@
-install: composer assets
+help: ## Display this help menu
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-composer:
+install: composer assets  ## Install PrestaShop
```

Comment:
The concept of installation is a bit vague here, we only install the dependencies, not prestashop itself.

Reply:
Oops it’s an error.